edgelessdb/getting-started/quickstart-azure.md

```diff
@@ -83,4 +83,6 @@ Now you can use EdgelessDB like any other SQL database:
 mysql -h<your-azure-ip> -uroot --ssl-ca edb.pem --ssl-cert cert.pem --ssl-key key.pem
 ```
 
+In case you skipped the verification step above, omit `--ssl-ca edb.pem` in the above command.
```

Comment:
```suggestion
In case you skipped the [verification step](#verify-your-edgelessdb-instance), omit `--ssl-ca edb.pem` in the `mysql` command.
```
(From the [Google documentation style guide](https://developers.google.com/style/translation#general-dos-and-donts)) Avoid directional language when possible.